src/SAML2/AuthnRequest.php

```diff
@@ -35,9 +38,9 @@ class AuthnRequest extends Request
     /**
      * Optional ProviderID attribute
      *
-     * @var string|null
+     * @var string
      */
-    private $ProviderName = null;
+    private $ProviderName;
```

Comment:
As far as I see in the specs, the `ProviderName` is optional, meaning this should default to `null`. Otherwise, we would always be adding a `ProviderName` even if we just have an empty string as the default.